sql/core/src/test/resources/sql-tests/results/cte.sql.out

```diff
@@ -98,16 +98,70 @@ struct<x:int>
 
 
 -- !query 8
-DROP VIEW IF EXISTS t
+WITH t(x, y) AS (SELECT 1, 2)
+SELECT * FROM t WHERE x = 1 AND y = 2
 -- !query 8 schema
-struct<>
+struct<x:int,y:int>
 -- !query 8 output
-
+1	2
 
 
 -- !query 9
-DROP VIEW IF EXISTS t2
+WITH t(x, x) AS (SELECT 1, 2)
+SELECT * FROM t
 -- !query 9 schema
-struct<>
+struct<x:int,x:int>
 -- !query 9 output
+1	2
+
+
+-- !query 10
+WITH t() AS (SELECT 1)
+SELECT * FROM t
+-- !query 10 schema
+struct<>
+-- !query 10 output
+org.apache.spark.sql.catalyst.parser.ParseException
+
+no viable alternative at input 'WITH t()'(line 1, pos 7)
```

Comment:
Do you think we can have a more meaningful message?

Reply:
I think this standard message comes from antlr when the query doesn't fit to the .g4 definition. I'm not sure if it can be fixed.